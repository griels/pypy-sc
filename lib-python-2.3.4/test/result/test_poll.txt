command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_poll.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Wed Apr 27 22:26:41 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_poll.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type 'select.poll'>
Traceback (application-level):
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_poll.py", line 171 in ?
test_poll1()
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_poll.py", line 37 in test_poll1
        p.register(rd, select.POLLIN)
AttributeError: 'poll' object has no attribute 'register'
========================== closed ==========================
execution time: 6.85833787918 seconds
exit status: 1
