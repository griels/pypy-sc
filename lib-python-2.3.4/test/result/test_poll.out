test_poll
Running poll test 1
